app/src/lib/parse-app-url.ts

```diff
@@ -102,10 +102,7 @@ export function parseAppURL(url: string): URLActionType {
   const parsedPath = pathName.substr(1)
 
   if (actionName === 'openrepo') {
-    const probablyAURL = parsedPath
-
-    // suffix the remote URL with `.git`, for backwards compatibility
```

Comment:
I'd prefer not to remove this line given the comment states this is for backwards compatibility.

Reply:
That line was added by @shiftkey during a refactor and the backwards compatibility is as far as I can tell referring to Desktop itself and not some ancient GHE and or GitHub.com version. I think @shiftkey added it more as a way of saying "I don't know if this is really necessary but it's been like this since forever so I'm not gonna deal with it in this refactor". Please correct me if I'm wrong @shiftkey.

In order words I think the comment itself is confusing as it could be interpreted as backwards compatibility with GHE/GitHub.com.

Reply:
I'm really hazy on the details but this probably came up because the URLs that were generated on GitHub looked like this:

```
x-github-client://openRepo/https://github.com/desktop/desktop
```

While we can get away with omitting these for GitHub, perhaps I was thinking about non-GitHub remotes that might also support these protocol handlers, but require a `.git` suffix?

Also note that this work pre-dates implementing the CLI that leverages this too, so I wasn't thinking about users being able to specify URLs with or without `.git`.